.vscode/settings.json

```diff
@@ -18,5 +18,8 @@
     "ui.semanticTokens": true,
     "formatting.gofumpt": true,
     "build.standaloneTags": ["ignore", "tools"]
+  },
+  "[go]": {
+    "editor.rulers": [111]
```

Comment:
Isn't the linter using `120`?

Reply:
Nope the linter `111` has the default length of 111 characters per line. 😅

The default ruler of VSCode is at character 80 which is misleading if the linter is at 111.

Reply:
Ok I re read the linter docs and the default should be at 120 as you mentioned.

I'm a bit confused why I get linter errors over 111. 🤨

Reply:
Ok it's getting late...I did read lll as 111 and got confused...🤦‍♂️

It's as you said 120.

I'll change the editor config.